From 010e6c5fe89059edff82fdbc05726e26e6a0b2b0 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Mon, 11 Mar 2019 16:38:32 +0000
Subject: [PATCH 627/725] staging: vc-sm-cma: Update TODO.

The driver is already a platform driver, so that can be
deleted from the TODO.
There are no known issues that need to be resolved.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 drivers/staging/vc04_services/vc-sm-cma/TODO | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/staging/vc04_services/vc-sm-cma/TODO
+++ b/drivers/staging/vc04_services/vc-sm-cma/TODO
@@ -1,2 +1 @@
-1) Convert to a platform driver.
-
+No currently outstanding tasks except some clean-up.
